lib/cli/commands/cancelorder.ts

```diff
@@ -2,18 +2,22 @@ import { callback, loadXudClient } from '../command';
 import { Arguments } from 'yargs';
 import { CancelOrderRequest } from '../../proto/xudrpc_pb';
 
-export const command = 'cancelorder <id>';
+export const command = 'cancelorder <id> <pair_id>';
 
 export const describe = 'cancel an order';
 
 export const builder = {
   id: {
```

Comment:
It might be helpful to call this `order_id` to further differentiate from `pair_id`.